src/RuleAction.php

```diff
@@ -49,6 +49,14 @@ public function getForbiddenStandardMassiveAction()
 
         $forbidden   = parent::getForbiddenStandardMassiveAction();
         $forbidden[] = 'update';
+
+        if (isset($_POST['rule_class_name'])) {
```

Comment:
a check on sub class of Rule maybe to avoid security issue ?

Reply:
I do not see the security risk here; but yes, if you want